### ISSUE TITLE
Add venue info/location info.

### ISSUE THREAD

**Issue by jadudm** [sigcsets/sigcse2017#5]
And the basic location/venue info?

**Comment by s-edwards** [comment]
Yes, please.  You can use "coming soon" as placeholders for the registration links.  Michael e-mailed links to pictures you can use.  The main hotel is the Sheraton Seattle http://www.sheratonseattle.com/.  Alternate hotels are the Grand Hyatt Seattle https://seattle.grand.hyatt.com/, Hyatt at Olive 8 https://olive8.hyatt.com/, and The Roosevelt Hotel http://roosevelthotel.com/.  Once we get reservation links for our group rate at these hotels, I'll open a separate issues to replace the "coming soon" placeholders.

**Comment by s-edwards** [comment]
BTW, venue is the Washington State Convention Center: http://www.wscc.com/.  Michael has some pics in the e-mail he sent.

**Comment by s-edwards** [comment]
Any update on this, and its related issues?  Possible to make the changes this weekend?
